manifests/b/Blueberry/FlashbackExpress/5.54.0.4702/Blueberry.FlashbackExpress.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Blueberry.FlashbackExpress
+PackageVersion: 5.54.0.4702
+PackageLocale: en-US
+Publisher: Blueberry Software (UK) Ltd.
+PublisherUrl: https://www.flashbackrecorder.com/
+PublisherSupportUrl: https://www.flashbackrecorder.com/support/
+PrivacyUrl: https://www.flashbackrecorder.com/privacypolicy/
+Author: Blueberry Software (UK) Ltd.
+PackageName: FlashBack Express 5

```

If this software is a replacement for FlashBack Express 1-5 we should use `FlashBack Express` as the PackageName, if they are different softwares and can be installed sideby side it should go under the PackageID `Blueberry.FlashbackExpress.5` as `FlashBack Express 5`
```suggestion
PackageName: FlashBack Express
```

---

That's completely fine.